api_food/foodapi.py

```diff
@@ -0,0 +1,43 @@
+#access the USDA food database, for Take a Byte
+#
+
+import requests #pip3 install requests
+import json 
+
+type = '/v1/foods/search'
```

Comment:
Redefining built-in 'type'Pylint[W0622:redefined-builtin](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/redefined-builtin.html)
Constant name "type" doesn't conform to UPPER_CASE naming stylePylint[C0103:invalid-name](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/invalid-name.html)

please fix